build.gradle

```diff
@@ -271,6 +297,9 @@ project(':iceberg-bundled-guava') {
 }
 
 project(':iceberg-api') {
+  test {
```

Comment:
Is there a way to configure all jvm args instead of individually?

Reply:
I tried a lot of options for ex : setting this in `org.gradle.jvmargs` or setting it in `DEFAULT_JVM_OPTS` or `GRADLE_OPTS` but the only way i could make it work was via above, I am open to all suggestions here.

Reply:
I was thinking if we could configure them all at allProjects or subPtojects hook https://github.com/apache/iceberg/blob/master/build.gradle#L200. If not, I cannot think of a better way either

Reply:
This works, except for integration test Task, thanks for the suggestion, have made the changes for remaining places.